app/assets/javascripts/sidebar.js

```diff
@@ -0,0 +1,7 @@
+
+$(document).ready(function() {
+  $('.time-period-select').select2();
+  $( ".time-period-select" ).change(function(event) {
```

Comment:
this line is not consistent with the line above. are we using single our double quotes in JS? Also there are extra spaces around the selector